source/support.tex

```diff
@@ -444,18 +498,43 @@
 \end{codeblock}
 
 \pnum
+For all members declared
+\tcode{static} \tcode{constexpr}
+in the
+\tcode{numeric_limits}
+template, specializations shall define these values in such a way
+that they are usable as
+constant expressions.
```

Comment:
See also [LWG 2833](https://cplusplus.github.io/LWG/lwg-active.html#2833) "Library needs to specify what it means when it declares a function constexpr".